homeassistant/components/prusalink/__init__.py

```diff
@@ -201,7 +201,7 @@ async def _fetch_data(self) -> JobInfo:
         return await self.api.get_job()
 
 
-class PrusaLinkEntity(CoordinatorEntity[PrusaLinkUpdateCoordinator]):  # pylint: disable=hass-enforce-coordinator-module
+class PrusaLinkEntity(CoordinatorEntity[PrusaLinkUpdateCoordinator]):
```

Comment:
The pylint check only enforces that classes derived from `DataUpdateCoordinator` should be placed inside a dedicated `coordinator.py` file - not from `CoordinatorEntity`.
https://github.com/home-assistant/core/blob/438db92d86e6dc1104a9153eb96d17161e12d1ad/pylint/plugins/hass_enforce_coordinator_module.py#L46-L50

Reply:
@CC @joostlek